src/libraries/Microsoft.PowerFx.Core/Public/DefinitionsCheckResult.cs

```diff
@@ -43,6 +51,7 @@ public class DefinitionsCheckResult : IOperationStatus
 
         public DefinitionsCheckResult()
         {
+            _localSymbolTable = new SymbolTable();
```

Comment:
>new SymbolTable(); [](http://example.com/codeflow?start=32&length=18)

nit, we can set DebugName here to aid in keeping symbol tables straight. 

new SymbolTable { DebugName = "FunctionDefinitions" }